_episodes/07-vector-data-in-python.md

```diff
@@ -10,8 +10,8 @@ objectives:
 - "Access the attributes of a spatial object with `geopandas`."
 keypoints:
 - "Vector dataset metadata include geometry type, CRS, and extent."
-- "Load spatial objects into Python with the `geopandas.read_file()` method."
-- "Spatial objects can be plotted directly with `geopandas.GeoDataFrame.plot()`."
+- "Load spatial objects into Python with `geopandas`'s `read_file()` function."
+- "Spatial objects can be plotted directly with `GeoDataFrame`'s `.plot()` method."
```

Comment:
I found this format:  `geopandas`'s `read_file()` a bit weird. Can we maybe use "the `geopandas.read_file() function`" instead?

Reply:
You are right - it was just that `geopandas` is always imported as `gpd` in the episode, but I am sure it is clear enough.